admin-dev/themes/new-theme/js/app/pages/stock/mixins/translate.ts

```diff
@@ -22,11 +22,11 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
-import Vue from 'vue';
+import {defineComponent} from 'vue';
 
-export default Vue.extend({
+export default defineComponent({
   methods: {
-    trans(key: string) {
+    trans(key: string): string {
```

Comment:
Is this a translatable string? What are the context and the domain, please? Can't find it on Crowdin.